500 cigarettes (New cigarette vendors and cigarette pack)

AlanParkerMasters opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->

# About the pull request

![cig](https://github.com/user-attachments/assets/8f499f58-0bda-42b3-820c-f157910d88a1)

Ports over some of this PR: [https://github.com/cmss13-devs/cmss13/pull/8204]
Adds two new vendors cigarette vendors: **Koorlander** and **W-Y**.
New pack of cigarettes: **Balaji** from the first Alien movie.
Replaces the majority of template cigarette vendors with Koorlander brand machines. 
More affluent areas (Command/W-Y) get W-Y vendors. 
Adds the unused UPP cigarette vendors to the Chapeav. 

While the PVP PR removed the template vendors from existence, I kept the template cigarette vendors since I like having some different vendors around maps to keep things a bit fresh, they are still identical to how they were, just a rare vendor now.
I also kept the original blue zippo lighter which was changed to 'Koorlander blue' since I thought the blue lighter looks cooler.


# Explain why it's good for the game
Variety is good.


# Testing Photographs and Procedure


# Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly label your changes in the changelog. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->
<!-- If you add a name after the ':cl', that name will be used in the changelog. You must add your CKEY after the CL if your GitHub name doesn't match. Maintainers freely reserve the right to remove and add tags should they deem it appropriate. -->

:cl:
add: Koorlander and W-Y cigarette vendors
add: Balaji cigarettes
maptweak: replaces most template cigarette vendors with Koorlander brand vendors, some with W-Y, and keeps basic cigarette vendors as a rarity
maptweak: USS Golden Arrow cafeteria vendors moved around a bit
maptweak: adds a single basic snack vendor to LV-624 maps for more diversity in the Nexus vendors
/:cl:

<!-- Both :cl:'s are required for the changelog to work! -->

AlanParkerMasters committed:
500 cigarettes

AlanParkerMasters committed:
fixes

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into 500_cigarettes

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
Fixup maps in TGM format

be54ff4ab187: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm
be54ff4ab187: maps/map_files/Ice_Colony_v3/Shivas_Snowball.dmm
be54ff4ab187: maps/map_files/LV624_Fixed/LV624_repaired.dmm
be54ff4ab187: maps/map_files/golden_arrow/golden_arrow.dmm

Automatically commited by: tools\mapmerge2\fixup.py

AndroBetel commented on code/game/machinery/vending/vending_types.dm:
```diff
@@ -269,6 +270,217 @@
 		/obj/item/tool/lighter/zippo = 25,
 
 	)
+	product_type = VENDOR_PRODUCT_TYPE_NICOTINE
```
this is inherited from parent class, no need to redefine it here and below

AlanParkerMasters replied:
Removed, thanks

AlanParkerMasters committed:
removes unnecessary redefines
